js/inputbar.js

```diff
@@ -294,6 +294,13 @@ weechat.directive('inputBar', function() {
                     return true;
                 }
 
+                if ($event.altKey && (code === 38 || code === 40)) {
```

Comment:
I would prefer it if this code also checked that ctrlkey wasn't down. As right now this would also work with ctrl-alt-arrow. In the future we might would want to do something different for that, or this might trigger something else in some other layout that we don't know about. And in general, it's better to be explicit if you mean alt+arrow and *only* alt-arrow, not all the other combinations.